Examples/Tests/initial_distribution/analysis_distribution.py

```diff
@@ -54,6 +49,10 @@
 print('Gaussian distribution difference:', f1_error)
 print('Maxwell-Boltzmann distribution difference:', f2_error)
 
+# print tolerance
+tolerance = 1.7e21
```

Comment:
Is `n     = 1.0e21` the peak density? If so, isn't `tolerance = 1.7e21` too big?
Also, could you do the comparison on relative tolerance, say `tolerance = 0.01`, that you would re-use for all tests?

Reply:
`n = 1.0e21` is not the peak density, but the density, the plasma is uniform. The peak value of `f` of the first test can be about 3.2e23. That's why I was using `max_to_unity` to make the tolerance close to 1 and all the same. I will try to compute a relative error then.

Reply:
I guess we can simply use the error divided by the peak value.

Reply:
Ah yes, it's the plasma density, sorry.